source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDateTextField.mm

```diff
@@ -53,8 +53,11 @@ - (instancetype)initWithTimeDateInput:(std::shared_ptr<BaseInputElement> const &
 
         picker.locale = [NSLocale currentLocale];
 
+        self.isAccessibilityElement = YES;
         self.id = [NSString stringWithCString:elem->GetId().c_str()
                                      encoding:NSUTF8StringEncoding];
+        self.accessibilityIdentifier = self.id;
+
```

Comment:
if this identifier is purely for identifying, please consider using tag, instead.

then you can get the view by doing [adaptive-card-view viewWithTag:tag];